src/Illuminate/Http/TransformableResponse.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+namespace Illuminate\Http;
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+use Illuminate\Validation\ValidationData;
+use Illuminate\Contracts\Support\Arrayable;
+
+abstract class TransformableResponse extends JsonResponse
+{
+    /**
+     * Constructor.
+     *
+     * @param  mixed  $data
+     * @param  int    $status
```

Comment:
Please use exactly 2 spaces before the dollar for param annotations on new code, rather than aligning.